Utility.cpp

```diff
@@ -95,14 +102,16 @@ void useAttackItem(Character*, Item* item)
     }
     else if( auto* ch = dynamic_cast<DragonSlayer*>(character))
     {
-        assert(false);
+        //assert(false);
         //DragonSlayers get a 10x boost when attacking dragons, from their attack item.
         //so their attack item should boost their attack damage by a factor of 10
         //this means you need to GET the attack damage, multiply it by the item's boost, and BOOST the attackDamage with that multiplied value.  
         //check Character.h for available member functions you can use.
+        
+        ch->boostAttackDamage(ch->getAttackDamage() * 10);
```

Comment:
you got the attack damage, you boosted the attackDamage, but you **didn't** get the item's boost. 
```
this means you need to GET the attack damage, multiply it by the item's boost, 
and BOOST the attackDamage with that multiplied value.  
```

- [ ] FIXME according to instructions.